packages/wp-now/src/tests/wp-now.spec.ts

```diff
@@ -144,3 +151,107 @@ test('isWordPressDevelopDirectory returns false for incomplete WordPress-develop
 	expect(isWordPressDevelopDirectory(projectPath)).toBe(false);
 	expect(inferMode(projectPath)).toBe(WPNowMode.INDEX);
 });
+
+describe('Test starting different modes', () => {
+	let tmpExampleDirectory;
+
+	/**
+	 * Copy example directory to a temporary directory
+	 *
+	 * Also, silent the console.log() output.
+	 */
+	beforeEach(() => {
+		const tmpDirectory = os.tmpdir();
+		const directoryHash = crypto.randomBytes(20).toString('hex');
```

Comment:
Let's add a `wp-now-tests-` prefix to the `directoryHash` so it's somewhat obvious where these directories are coming from.

Reply:
Good idea, I've added the flag. Note that we are removing the directory in `afterEach`, so ideally they shouldn't stay in `/tmp` after tests are executed. It is still a good change for cases in which the test crashes and doesn't run the cleanup method.